dtfs-central-api/src/helpers/amendments/tfm-facility-latest-completed-amendment-to-facility-value.ts

```diff
@@ -0,0 +1,21 @@
+import { TfmFacility } from '@ukef/dtfs2-common';
+import orderBy from 'lodash.orderby';
+import { getCompletedAmendments } from './completed-amendments';
+
+/**
+ * Get the latest completed facility value amendment
+ * @param tfmFacility - The tfm facility
+ * @returns The latest completed facility value amendment
+ */
+export const getLatestCompletedAmendmentToFacilityValue = (tfmFacility: TfmFacility): number | undefined => {
+  const { amendments } = tfmFacility;
+  if (!amendments) {
+    return undefined;
+  }
+
+  const completedAmendments = getCompletedAmendments(amendments);
+
+  const latestAmendmentWithDefinedFacilityValue = orderBy(completedAmendments, ['updatedAt'], ['desc']).find(({ value }) => value);
```

Comment:
```suggestion
  const latestAmendmentsWithFacilityValues = orderBy(completedAmendments, ['updatedAt'], ['desc']).find(({ value }) => value);
```

Reply:
I will remove the word defined, but it returns just a single amendment and amendments have a single facility value so I am not adding the pluralisations.

Reply:
Sorry I can see the full context now with this being called in a map for each fee record,  makes sense 👍